py/client/pydeephaven/updateby.py

```diff
@@ -626,23 +624,25 @@ def rolling_sum_time(ts_col: str, cols: Union[str, List[str]], rev_time: int,
 
     Here are some examples of window values:
         rev_time = 0, fwd_time = 0 - contains rows that exactly match the current row timestamp
-        rev_time = 600_000_000_000, fwd_time = 0 - contains rows from 10m before through the current row timestamp (
+        rev_time = "PT00:10:00", fwd_time = "0" - contains rows from 10m before through the current row timestamp (
             inclusive)
         rev_time = 0, fwd_time = 600_000_000_000 - contains rows from the current row through 10m following the
             current row timestamp (inclusive)
-        rev_time = 600_000_000_000, fwd_time = 600_000_000_000 - contains rows from 10m before through 10m following
+        rev_time = "PT00:10:00", fwd_time = "PT00:10:00" - contains rows from 10m before through 10m following
             the current row timestamp (inclusive)
-        rev_time = 600_000_000_000, fwd_time = -300_000_000_000 - contains rows from 10m before through 5m before the
+        rev_time = "PT00:10:00", fwd_time = "-PT00:05:00" - contains rows from 10m before through 5m before the
             current row timestamp (inclusive), this is a purely backwards looking window
-        rev_time = -300_000_000_000, fwd_time = 600_000_000_000 - contains rows from 5m following through 10m
+        rev_time = "-PT00:05:00", fwd_time = "PT00:10:00"} - contains rows from 5m following through 10m
             following the current row timestamp (inclusive), this is a purely forwards looking window
-
+    
     Args:
         ts_col (str): the timestamp column for determining the window
         cols (Union[str, List[str]]): the column(s) to be operated on, can include expressions to rename the output,
             i.e. "new_col = col"; when empty, update_by perform the rolling sum operation on all columns.
-        rev_time (int): the look-behind window size, in nanoseconds
-        fwd_time (int): the look-behind window size, in nanoseconds
+        rev_time (int): the look-behind window size, can be expressed as an integer in nanoseconds or a time
+            interval string, e.g. "PT00:00:.001" or "PT5M"
+        fwd_time (int): the look-ahead window size, can be expressed as an integer in nanoseconds or a time
+            interval string, e.g. "PT00:00:.001" or "PT5M", default is 0
```

Comment:
docs need the types updated.  All places in the doc strings need to be checked.  I just noticed it here.